### PR TITLE
feat: add boltz icon to QR code

### PR DESCRIPTION
**Pull request by dni**
refactor qrcodes into `QrCode` component.

screenshot-1700483623

### CODE REVIEW

File: src/components/QrCode.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import "../style/qrcode.scss";
+
+export const Qrcode = ({ data }) => {
+    const [dataUrl, setDataUrl] = createSignal("");
+
+    QRCode.toDataURL(data, { width: 300 })
+        .then(setDataUrl)
+        .catch((err: Error) => {
+            log.error("qr code generation error", err);
+        });
```
Comment (michael1011) [inline src/components/QrCode.tsx:14]: This should be in a `createEffect` call to make sure it's updated when `data` changes

**Review by michael1011 — CHANGES_REQUESTED** [review]

**Comment by kilrau** [comment]
Looks good, scanning works. From a UX test side approved. I like it!

**Review by michael1011 — CHANGES_REQUESTED** [review]
```
Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($size, 2) or calc($size / 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

  ╷
9 │         margin-left: -($size/2);
  │                        ^^^^^^^
  ╵
    src/style/qrcode.scss 9:24  root stylesheet
   ```

And there is a deprecation warning in the new SCSS. Wrapping the div in `calc` seems to work

**Comment by dni** [comment]
done

**Review by kilrau — APPROVED** [review]

File: src/components/QrCode.tsx (hunk 1 above)
Comment (kilrau) [reply src/components/QrCode.tsx:14]: Agree, an invoice is fix for one swap id. Not sure what you mean @michael1011 , please ack that this can be merged

**Comment by dni** [comment]
@michael1011 like so?

**Pull request merged**